src/Bolt/ProtocolFactory.php

```diff
@@ -32,7 +32,11 @@ class ProtocolFactory
      */
     public function createProtocol(IConnection $connection, AuthenticateInterface $auth, string $userAgent): array
     {
-        putenv('BOLT_ANALYTICS_OPTOUT=1');
+        $boltOptoutEnv = getenv('BOLT_ANALYTICS_OPTOUT');
+        /** @psalm-suppress RiskyTruthyFalsyComparison */
+        if (empty($boltOptoutEnv)) {
```

Comment:
it should be `$boltOptoutEnv === false`

because in the case env variable is set to `0`, then empty will return true.